meta_tools/templates/template_config_sweep.py

```diff
@@ -0,0 +1,52 @@
+from utils import utils_script_gen
+from pathlib import Path
+
+
+def generate(script_dir: Path, model_architecture: str) -> bool:
```

Comment:
For generating sweep scripts, stepshiter models use a slightly different template, e.g., a mandatory key called 'name' (model name), no model architecture. I suggest leaving parameters_dict empty to adapt to different model templates.

Reply:
Agreed